test/plain-list/AutosuggestApp.test.js

```diff
@@ -131,6 +132,83 @@ describe('Plain list Autosuggest', () => {
     });
   });
 
+  describe('when typing and matches exist (when selectFirstSuggestion is true)', () => {
+    before(() => {
+      selectFirstSuggestion.true();
+    });
+
+    after(() => {
+      selectFirstSuggestion.false();
```

Comment:
Yeah I wasn't crazy about it either. I was trying to mirror the semantics of the `spy.reset()` calls. I'll update it.